testing-unit-common/src/main/java/com/bumble/appyx/testing/unit/common/util/InteropSimpleBuilderStub.kt

```diff
@@ -0,0 +1,31 @@
+package com.bumble.appyx.testing.unit.common.util
+
+import androidx.lifecycle.Lifecycle
+import com.bumble.appyx.core.builder.SimpleBuilder
+import com.bumble.appyx.core.modality.BuildContext
+import com.bumble.appyx.core.node.Node
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class InteropSimpleBuilderStub(
+    val delegate: (BuildContext) -> Node = { _ -> NodeStub() },
+) : SimpleBuilder() {
+
+    private val _createdNodes = ArrayList<Node>()
```

Comment:
At the moment does this need to be a list?

Reply:
Not really, I changed it in 592c9d9268cfd05127ea0c4739787a7430fd4fcc